### ISSUE TITLE
I can not manage to recieve notification

### ISSUE THREAD

**Issue by muratdogan17** [OneSignal/react-native-onesignal#397]
Hi, I've set up all the things as it has been written on readme. I get a confirm pop up to allow notifications. I see the device which I have allowed on OneSignal's site. when I try to make a push notification, it fails. Nothing happens on my device which I debug my app. What should I do and where should I check?

"dependencies": {
		"react": "16.0.0",
		"react-native": "0.50.1",
		"react-native-onesignal": "^3.0.9"
	},

**Comment by lfaz** [comment]
I am facing somehow same issue but only after publishing the app, for me in debug mode it works perfectly fone for both android and ios when using exp start --lan, but when I publish the app using exp publish then I am no longer able to receive push notifcation.

**Comment by Nightsd01** [comment]
@lulzimfazlija this would not be an sdk issue if it works in development but not production, that would be a certificate issue. 

https://documentation.onesignal.com/docs/generate-an-ios-push-certificate

**Comment by Nightsd01** [comment]
@muratdogan17 can you provide some more details? When you say “it fails”, what do you mean? How are you sending the push notification, through the onesignal dashboard?

Also, what kind of device are you testing on, android or iOS? Simulator or a real device?

**Comment by muratdogan17** [comment]
@Nightsd01 I mean it says Invalid when I try to send push. I'm sending through the onesignal dashboard.

I'm testing on real device, ios iphone 7. I'm trying both debugging on xcode using real device and testflight with new version.

**Comment by Nightsd01** [comment]
@muratdogan17 Can you copy+paste the message you’re getting?

**Comment by muratdogan17** [comment]
@Nightsd01 actually, I do not have any meaningful software error message it just says Invalid under the Delivery Statistics and list 3 possible reasons which are : 

> These errors are typically caused due to one of the following reasons:
> 
>     Your OneSignal Application Settings are incorrect.
>     These tokens belong to an app that does not match your OneSignal settings.
>     Some other backend error occured.

**Comment by lfaz** [comment]
@Nightsd01 I have already done that part, i mean i followed the link you have providen, still does not work! quite weird, also it is the same case for android! all works perfectly fine on debug mode both, but both not working on production!

**Comment by Nightsd01** [comment]
@lulzimfazlija the SDK doesn't know/care if it is debug or dev. @muratdogan17 that is definitely an app settings issue and wouldn't be caused by the SDK. This would definitely be a certificate error in both of your cases. I would suggest contacting OneSignal support by going to https://www.onesignal.com and clicking the ? button at the top right.

**Comment by lfaz** [comment]
@Nightsd01 it works if I am building the apk from Android Studio Signed APK. thnx!
